tests/test_recipe/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "feedstock_debug_test" %}
```

Comment:
Test meta.yaml is at two places. Please remove the one from tests dir, keeping this one.

Reply:
Actually, this meta.yaml is not even needed and read as we have mocked debug api.